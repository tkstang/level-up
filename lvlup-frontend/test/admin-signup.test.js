import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import thunk from 'redux-thunk';
import configureMockStore from 'redux-mock-store';
import AdminSignUp from '../src/components/Admin/signup/signup';
import lvlupApp from '../src/reducers/index';

const middlewares = [thunk];
const mockStore = configureMockStore(middlewares);

test('adminsignup should render', () => {
  const store = mockStore({ lvlupApp });
  const adminsignup = shallow(
    <AdminSignUp adminSignup={{ status: false }} store={store} />,
  );
  expect(shallowToJson(adminsignup)).toMatchSnapshot();
});
