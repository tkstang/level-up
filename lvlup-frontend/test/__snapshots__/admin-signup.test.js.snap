exports[`test adminsignup should render 1`] = `
<div>
  <div
    className="signup">
    <Connect(lifecycle(ReduxForm)) />
  </div>
</div>
`;
